### PR TITLE
Upgrade the Rails version to 7.1.3 for the lumberaxe library.

### PR DESCRIPTION
**Pull request by rurkss**
To address the [dependency conflict](https://github.com/powerhome/example-rails-app/pull/335) in the `example-rails-app` application, the team @powerhome/forever-people proposes upgrading the [rails gem to version 7.1.3](https://rubygems.org/gems/rails).

### CODE REVIEW

File: packages/lumberaxe/lumberaxe.gemspec [hunk 1]
```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "parser", ">= 2.5", "!= 2.5.1.1"
   spec.add_development_dependency "pry", ">= 0.14.2"
   spec.add_development_dependency "pry-byebug", "3.10.1"
-  spec.add_development_dependency "rails", ">= 6.0.6.1", "< 7.1"
+  spec.add_development_dependency "rails", ">= 6.0.6.1", "<= 7.1.3"
```
Comment (benlangfeld) [inline packages/lumberaxe/lumberaxe.gemspec:43]:
```suggestion
  spec.add_development_dependency "rails", ">= 6.0.6.1", "<= 7.1"
```

Let's not be specific about patch versions.

File: packages/lumberaxe/lumberaxe.gemspec [hunk 2]
```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", ">= 6.0.6.1", "< 7.1"
+  spec.add_dependency "activesupport", "7.1.3"
```
Comment (benlangfeld) [inline packages/lumberaxe/lumberaxe.gemspec:33]: This library should not force a specific Rails version on its users.

```suggestion
  spec.add_dependency "activesupport", ">= 6.0.6.1", "<= 7.1"
```

File: packages/lumberaxe/Gemfile.lock [hunk 3]
```diff
@@ -12,99 +12,117 @@ PATH
   remote: .
   specs:
     lumberaxe (0.1.4)
-      activesupport (>= 6.0.6.1, < 7.1)
+      activesupport (= 7.1.3)
```
Comment (xjunior) [inline packages/lumberaxe/Gemfile.lock:15]: This should be the same as the gemspec, `"activesupport", ">= 6.0.6.1", "<= 7.1"`

**Review by xjunior — APPROVED** [review]

**Comment by benlangfeld** [comment]
Auto merge is dangerous when your CI checks are not required.

**Pull request merged**